tasks/main.yml

```diff
@@ -18,7 +25,11 @@
     dest: '/etc/iscsi/initiatorname.iscsi'
     regexp: '^InitiatorName='
     line: "InitiatorName={{ custom_iscsi_initiator_name is defined|ternary(custom_iscsi_initiator_name,iscsi_initiator_name) }}"
-  notify: restart iscsid
+    create: 'yes'
+    owner: 'root'
+    group: 'root'
+    mode: '0600'
```

Comment:
**[?]** Is there reason why the this file should be created? On which OS/package version this file was not present so  `create: yes` option was needed?

Reply:
I had a case where I removed /etc/iscsi, reinstalled all packages and that file was somehow missing.
Might not be needed too often, but makes this less errorprone.

Reply:
As I don't see any nice way how to force reinstallation of package that would bring this back reliably lets keep this.